deploy/charts/trust-manager/README.md

```diff
@@ -22,50 +22,877 @@ Kubernetes: `>= 1.25.0-0`
-| defaultPackageImage.tag | string | `"20210119.0"` | Tag for the default package image |
-| filterExpiredCertificates.enabled | bool | `false` | Whether to filter expired certificates from the trust bundle. |
-| image.digest | string | `nil` | Target image digest. Will override any tag if set. for example: digest: sha256:0e072dddd1f7f8fc8909a2ca6f65e76c5f0d2fcfb8be47935ae3457e8bbceb20 |
-| image.pullPolicy | string | `"IfNotPresent"` | Kubernetes imagePullPolicy on Deployment. |
-| image.registry | string | `nil` | Target image registry. Will be prepended to the target image repositry if set. |
-| image.repository | string | `"quay.io/jetstack/trust-manager"` | Target image repository. |
-| image.tag | string | `nil` | Target image version tag. Defaults to the chart's appVersion. |
-| imagePullSecrets | list | `[]` | For Private docker registries, authentication is needed. Registry secrets are applied to the service account |
-| namespace | string | `""` | The namespace to install trust-manager into. If not set, the namespace of the release will be used. This is helpful when installing trust-manager as a chart dependency (sub chart) |
-| nodeSelector | object | `{"kubernetes.io/os":"linux"}` | Configure the nodeSelector; defaults to any Linux node (trust-manager doesn't support Windows nodes) |
-| priorityClassName | string | `""` | Configure the priority class of the pod; see https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass |
-| replicaCount | int | `1` | Number of replicas of trust-manager to run. |
-| resources | object | `{}` |  |
-| secretTargets.authorizedSecrets | list | `[]` | A list of secret names which trust-manager will be permitted to read and write across all namespaces. These will be the only allowable Secrets that can be used as targets. If the list is empty (and authorizedSecretsAll is false), trust-manager will not be able to write to secrets and will only be able to read secrets in the trust namespace for use as sources. |
-| secretTargets.authorizedSecretsAll | bool | `false` | If set to true, grant read/write permission to all secrets across the cluster. Use with caution! If set, ignores the authorizedSecrets list. |
-| secretTargets.enabled | bool | `false` | If set to true, enable writing trust bundles to Kubernetes Secrets as a target. trust-manager can only write to secrets which are explicitly allowed via either authorizedSecrets or authorizedSecretsAll. NOTE: Enabling secret targets will grant trust-manager read access to all secrets in the cluster. |
-| tolerations | list | `[]` | List of Kubernetes Tolerations; see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#toleration-v1-core |
-| topologySpreadConstraints | list | `[]` | List of Kubernetes TopologySpreadConstraints; see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#topologyspreadconstraint-v1-core |
+<!-- AUTO-GENERATED -->
```

Comment:
@ThatsMrTalbot Do you think adding sections could improve the readability of this file?

Reply:
Do we do this in a separate PR or do we do it here?